docs/source/about.rst

```diff
@@ -0,0 +1,6 @@
+What is Mythril Classic?
+========================
+
+Mythril Classic is a security analysis tool for Ethereum smart contracts. It was `introduced at HITBSecConf 2018 <https://github.com/b-mueller/smashing-smart-contracts/blob/master/smashing-smart-contracts-1of1.pdf>`_.
+
+Mythril Classic detects a range of security issues, including integer underflows, owner-overwrite-to-Ether-withdrawal, and others. However, the analysis will not detect business logic issues and is not equivalent to formal verification.
```

Comment:
`and is not equivalent to formal verification`
I think this is a bit ambiguous. Symbolic execution is a formal verification technique, but it isn't sound in most practical cases (which is what is relevant here)
I'd like to rephrase it as follows:
```suggestion
Mythril Classic detects a range of security issues, including integer underflows, owner-overwrite-to-Ether-withdrawal, and others. 
Note that Mythril is targeted at finding common vulnerabilities, and is not able to discover issues in the business logic of an application.
Furthermore, Mythril and symbolic executors, are generally unsound, as they are often unable to explore all possible states of a program.
```